docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -195,6 +213,15 @@ And, the specific default value is mandatory depending on each support types:
 | `<time-type>` | `<default-hour>` | `<default-short-text>` |
 |               | `<default-minute>` | `<default-extra-data>` |
 |               | `<default-second>` |         |
+|               | `<default-timezone-id>` |         |
+
+
+> **Note**
+   >
+   > `<default-hour>`, `<default-hour>`, `<default-hour>` are deprecated since Tizen 5.5. <br/>
+   Instead, use `<default-timezone-id>`. <br/>
+   `<default-timezone-id>` is the value that declared in TZ database. ex) Asia/Seoul <br/>
+   If the xml contains`<default-timezone-id>`, then `<default-hour>`, `<default-hour>`, `<default-hour>` are not necessary.

```

Comment:
Please make this minor change:

If the xml contains`<default-timezone-id>`, then `<default-hour>`, `<default-hour>`, and `<default-hour>` are not necessary.